components/dialogs/b2b/AccountHierarchyAddFormDialog/AccountHierarchyAddFormDialog.tsx

```diff
@@ -1,20 +1,20 @@
 import { useTranslation } from 'next-i18next'
 
-import { AccountHierarchyForm } from '@/components/b2b'
+import { AccountHierarchyAddForm } from '@/components/b2b'
 import { KiboDialog } from '@/components/common'
 import { CreateCustomerB2bAccountParams } from '@/lib/types'
 
 import { B2BAccount } from '@/lib/gql/types'
 
-interface AccountHierarchyFormDialogProps {
+interface AccountHierarchyAddFormDialogProps {
   accounts?: B2BAccount[]
   isAddingAccountToChild: boolean
   formTitle?: string
   onSave: (data: CreateCustomerB2bAccountParams) => void
   onClose: () => void
 }
 
-const AccountHierarchyFormDialog = (props: AccountHierarchyFormDialogProps) => {
+const AccountHierarchyAddFormDialog = (props: AccountHierarchyAddFormDialogProps) => {
```

Comment:
why custom type? can we use  B2BAccount?

Reply:
Not required as per our discussion over the call.
This type is used for all the values its accepting.